docs/en/oas3/OAS3-PDND-AS.yaml

```diff
@@ -206,8 +206,263 @@ paths:
               example:
                 error: "temporarily_unavailable"
                 error_description: "The request cannot be fulfilled because the e-Service Endpoint is temporarily unavailable (e.g., due to maintainance or overload)"
+  /v0.9.0/CryptographicInformations{eserviceId}:
+    get:
+      tags:
+        - e-Services PDND
+      summary: Get Cryptographic Informations
+      description: >-
+        This service provides the Credential Issuer with the pseudonymization algorithm and salt used to pseudonymize the Signal subject throughout the e-Services in Signal Hub.
+      operationId: CryptographicInformations
+      parameters:
+        - name: eserviceId
+          schema:
+            type: string
+            example: b1817321-0486-4c75-89e5-4ee297250418         
+          required: true
+          in: path
+          description: e-Service for which the pseudonymization algorithms is requested. It MUST correspond to the e-service Id value the Authentic Source is a Provider of.
+        - name: Authorization
+          in: header
+          description: >-
+            JWT token obtained from <a target="blank"
+            href="https://italia.github.io/eid-wallet-it-docs/v1.0.0/en/e-service-pdnd.html#voucher-issuance">PDND
```

Comment:
is there any impacts in using v1.0.0 without alignments to 1.1.0, 1.2.0 and so on?